bin/build-docker.sh

```diff
@@ -1,5 +1,7 @@
 #!/usr/bin/env bash
 
+# is this used?
```

Comment:
@areed i couldn't find any references to this file.

Reply:
It was used for kurl-util image but that has its own directory and makefile now